src/main/webapp/dev/js/main/userMap.es6

```diff
@@ -41,11 +41,11 @@ function initializeMap(err, countryCoordinates, userData) {
     });
 
     // set the last updated date in the page
-    $('#lastUpdate').get(0).innerHTML = date;
+    $('#lastUpdate').html() = date;
```

Comment:
Does this actually work? Usually it's used this way:
```javascript
$('#lastUpdate').html(date);
```
Same for the other instances.